apis/datasciencecluster/v1alpha1/datasciencecluster_types.go

```diff
@@ -69,6 +69,12 @@ type Serving struct {
 	TestConfigMap corev1.ConfigMap `json:"testConfigMap,omitempty"`
 }
 
+type Workbenches struct {
+	Component `json:""`
+	// List of configurable controllers/deployments
+	ManagedImages bool `json:"managedImages,omitempty"`
+}
+
```

Comment:
This code has been moved to a different file. This will require rebasing and re-doing the PR. https://github.com/VaishnaviHire/opendatahub-operator/blob/add_component_deployment/components/workbenches/workbenches.go#L17